Library/Formula/libgig.rb

```diff
@@ -0,0 +1,35 @@
+class Libgig < Formula
+  desc "library for Gigasampler and DLS (Downloadable Sounds) Level 1/2 files"
+  homepage "http://www.linuxsampler.org/libgig/"
+  url "http://download.linuxsampler.org/packages/libgig-3.3.0.tar.bz2"
+  sha256 "d22a3c7ba13d920c1d4b6c218107ad105622ae9d1236ffbce007b98547774425"
+
+  depends_on "pkg-config" => :build
+  depends_on "libsndfile"
+
+  def install
+    ENV.deparallelize
```

Comment:
Can you add a comment noting that? I'd also probably rather we used the latest version here.